packages/cardpay-cli/prepaid-card-market-v-2/add-tokens.ts

```diff
@@ -0,0 +1,47 @@
+import { Argv } from 'yargs';
+import { getEthereumClients, NETWORK_OPTION_LAYER_2, getConnectionType } from '../utils';
+import { Arguments, CommandModule } from 'yargs';
+import { getConstant, getSDK } from '@cardstack/cardpay-sdk';
+
+export default {
+  command: 'add-tokens <issuerSafe> <issuer> <token> <amount>',
+  describe: 'Add tokens to the contract',
+  builder(yargs: Argv) {
+    return yargs
+      .positional('issuerSafe', {
+        type: 'string',
+        description: 'The safe address used to fund the transfer to the contract',
+      })
+      .positional('issuer', {
+        type: 'string',
+        description: 'The address of the issuer',
+      })
```

Comment:
I was a bit confused by the `issuer` positional, I learnt through trial and error that this is an EOA. Does it have to be the EOA that owns the `issuerSafe`, and do the funds come from it or from the `issuerSafe`?

Reply:
Yes, the issuer is an EOA, and yes it has to be the same EOA that owns the issuer safe. The funds come from the issuer safe though. The idea is that the issuer can find their inventory from multiple safes but still have a stable SKU